cmd/bblfsh/client.go

```diff
@@ -15,6 +15,11 @@ import (
 	"github.com/Sirupsen/logrus"
 	"github.com/bblfsh/sdk/protocol"
 	"google.golang.org/grpc"
+	"srcd.works/go-errors.v0"
+)
+
+var (
+	errProfiler = errors.NewKind("Failed to create a % file at %s")
```

Comment:
Well, yes, I you are right, if someone was to import it - it would make sense to expose possible recoverable error conditions. And because it's not really an error condition at all, previous impl did not pass it out of the function.

I can hardly imagine anyone willing to import `main` package as dependency (IDK if that is possible), but as soon as we propagate this error, I guess you are right and we should expose it.